### DISCUSSION TITLE
Only running abinet appears ValueError: Blur limit must be odd when centered=True. Got: (3, 6)

### DISCUSSION THREAD

**Discussion by OuXiao514** [open-mmlab/mmocr#1794] [General]
How to solve?

**Comment by gaotongxiao** [comment]
It's because `albumentation` has restricted `blur_limit` to be an odd number in its later release. You can change it from 6 to 7:

https://github.com/open-mmlab/mmocr/blob/bfb36d81b3c87a9e4bb04832d8e9d813e55e07a3/configs/textrecog/abinet/_base_abinet-vision.py#L89

If you have time, could you raise a PR and help us fix it? Your contribution will be greatly appreciated.
